src/core/devutils/debughandlers/resizedebughandler/ResizeDebugConfig.ts

```diff
@@ -0,0 +1,8 @@
+import { Dimension } from "../../../..";
+import DebugConfig from "../DebugConfig";
+
+export default interface ResizeDebugConfig extends DebugConfig {
+    relaxation: Dimension;
+    onRelaxationViolation: (expectedDim: Dimension, actualDim: Dimension, index: number) => void;
+    forceEnable?: boolean;
```

Comment:
we don't need `forceEnable?: boolean;` as its already present in `DebugConfig`

Reply:
I am planning to remove this only. If we are taking from the client, we don't need to force override it.